services/tasks/templates/slack.tmpl

```diff
@@ -13,7 +13,14 @@
                     "title": "Author",
                     "value": "{{ .Author }}",
                     "short": true
+                },
```

Comment:
The comma should only be placed if a version is given

Reply:
I have made changes as suggested and tested it.

![image](https://github.com/semaphoreui/semaphore/assets/19252498/e18be37f-e3c9-447e-98c4-21c59d4d53df)